page-upskill.php

```diff
@@ -7,36 +7,35 @@
 
 <?php get_header(); ?>
 
-  <div class='main-section'>
+  <div class="main-section">
     <?php 
-      if ( have_posts() ) :
-    ?>
-        <ul class="posts">    
-      <?php
-        while ( have_posts() ) : the_post();
+      if ( have_posts() ) : ?>
+      	<ul class="posts">           
+      <?php 
+        while ( have_posts() ) : the_post(); 
       ?>
           <li>
-            <?php the_content(); ?>
-          </li>
-        </ul>
-      <?php
-          get_template_part( 'content', 'page' );
-      ?>
-          <article id="post-<?php the_ID(); ?>" <?php post_class(); ?>>
-
-          <?php the_title('<h1 class="entry-title">','</h1>' ); ?>
+            <?php the_title('<h1 class="entry-title">','</h1>' ); ?>
+            <article <?php post_class(); ?>>
+            <?php get_template_part( 'content', 'page' ); ?>
 
     <?php if( has_post_thumbnail() ): ?>
             <div class="thumbnails">
-              <?php the_post_thumbnail('thumbnail'); ?>
+                <?php the_post_thumbnail('thumbnail'); ?>
             </div>
     <?php endif; ?>
 
-          <small>
-            <?php the_category(); ?>
-          </small>
-    <?php 
-			  endwhile;
+            <small>
+              <?php the_category(); ?>
+            </small>
+
+            <?php the_content(); ?>
+          </li>
+      <?php
+        endwhile;
+      ?>
+        </ul>
```

Comment:
You only need a list if you display multiple posts, which the main loop on a page won't have.

There's two parts to this whole story:
- Formatting your html and php correctly
- Knowing when to use which elements